### DISCUSSION TITLE
Disable syslog. Setting `USE_SYSLOG=false` isn't working with pre-build binary + systemd service

### DISCUSSION THREAD

**Discussion by subhamagr** [dani-garcia/vaultwarden#1760] [Q&A] [answered]
I have my `.env` file:
```
...

# Logging
USE_SYSLOG=false
EXTENDED_LOGGING=true
ROCKET_CLI_COLORS=off
LOG_FILE=/var/log/vaultwarden/app.log

...
```

Setting `USE_SYSLOG=false` has absolutely no effect. I see logs in both syslog and `/var/log/vaultwarden/app.log`.

I have also tried setting `ExecStart=/vaultwarden > /dev/null 2>&1` in my `vaultwarden.service` file. Still same issue. 

Am I missing something?

**Comment by BlackDex** [comment, marked as answer]
I think you need to change the vaultwarden systemd config to contain the following:
```ini
[Service]
StandardOutput=null
StandardError=null
```

**Reply by subhamagr** [reply]
oh yes, thank you! 😅

**Reply by subhamagr** [reply]
@BlackDex btw, what's the use of `USE_SYSLOG`? I see that by default it's `false`.

**Reply by BlackDex** [reply]
you can disable logging to a log-file but still have logging to syslog.
This is mostly when not using systemd :).
